cfme/control/explorer.py

```diff
@@ -370,14 +370,14 @@ def f(_):
         "provider_alert_profile": _ap_single_branch("provider", "Provider"),
         "provider_alert_profiles": _ap_multi_branch("provider", "Provider"),
 
-        "host_alert_profile": _ap_single_branch("host", "Host"),
-        "host_alert_profiles": _ap_multi_branch("host", "Host"),
+        "host_alert_profile": _ap_single_branch("host", "Host / Node"),
+        "host_alert_profiles": _ap_multi_branch("host", "Host / Node"),
 
         "datastore_alert_profile": _ap_single_branch("datastore", "Datastore"),
         "datastore_alert_profiles": _ap_multi_branch("datastore", "Datastore"),
 
-        "cluster_alert_profile": _ap_single_branch("cluster", "Cluster"),
-        "cluster_alert_profiles": _ap_multi_branch("cluster", "Cluster"),
+        "cluster_alert_profile": _ap_single_branch("cluster", "Cluster / Deployment Role"),
+        "cluster_alert_profiles": _ap_multi_branch("cluster", "Cluster / Deployment Role"),
```

Comment:
We might want to go with `version.pick` here instead of deleting old locators. Discussed offline.